app/screens/thread/thread.tsx

```diff
@@ -80,7 +81,10 @@ const Thread = ({
     }, [componentId, rootId, isCRTEnabled]);
 
     useEffect(() => {
+        storeLastViewedThreadIdAndServer(rootId);
+
         return () => {
+            removeLastViewedThreadIdAndServer();
```

Comment:
question: is this true regardless of CRT being enabled or disabled?

Reply:
yeah